Timefy/Common/Extensions/UIColor+Extension.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  UIColor+Extension.swift
+//  Timefy
+//
+//  Created by Doğancan Mavideniz on 15.09.2022.
+//
+
+import UIKit
+extension UIColor {
+    class var appWhite: UIColor {
```

Comment:
class var kullanmanın özel bir sebebi var mı? Class var yapıldığında override edilebiliyor. Ama bu Tek bir yerden yönetilen renk olayını bozabilir

https://www.hackingwithswift.com/example-code/language/whats-the-difference-between-a-static-variable-and-a-class-variable